custom_components/mammotion/const.py

```diff
@@ -8,3 +8,4 @@
 DEFAULT_RETRY_COUNT = 3
 CONF_RETRY_COUNT = "retry_count"
 LOGGER: Final = logging.getLogger(__package__)
+SCAN_INTERVAL = timedelta(seconds=30)
```

Comment:
Somewhere between 1-5 minutes is more appropriate, since bleak is a shared resource